authservice_app.js

```diff
@@ -82,71 +112,70 @@ function initDB(){
 
 
 function startServer() {
-	if (serverStarted ) return;
+	if (serverStarted) return;
 	serverStarted = true;
-	app.listen(port);
+	fastify.listen(port, function (err) {
```

Comment:
I believe it could be done in an async/await try/catch manner to match more #3 requirements